guilford_sent_component.yaml

```diff
@@ -0,0 +1,597 @@
+          hashFunction: "sha256"
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "DATE OFFENSE COMMITTED - BEGIN"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+      ol.datetimeend:
+        type: "ol.datetimeend"
+        column: "DATE OFFENSE COMMITTED - END"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+    name: "chargeevent"
+
+  location:
+    fqn: "ol.location"
+    entitySetName: "GuilfordLocations"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "COUNTY OF CONVICTION CODE"
+      ol.county:
+        type: "ol.county"
+        column: "COUNTY OF CONVICTION CODE"
+    name: "location"
+
+  sentenceterm:
+    fqn: "ol.sentenceterm"
+    entitySetName: "GuilfordSentenceTerms_DPS"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "SENTENCE EFFECTIVE(BEGIN) DATE"] #just these two columns in order to overlap with sent_computation table
+      ol.minimumvalue:
+        type: "ol.minimumvalue"
+        column: "MINIMUM SENTENCE LENGTH"
+        transforms:
+        - !<transforms.ParseDoubleTransform> {}
+      ol.maximumvalue:
+        type: "ol.maximumvalue"
+        column: "MAXIMUM SENTENCE LENGTH"
+        transforms:
+        - !<transforms.ParseDoubleTransform> {}
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "SENTENCE EFFECTIVE(BEGIN) DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+    name: "sentenceterm"
+
+  jailtimemodifications:
```

Comment:
CAn we add either property ol.type or ol.description to this, and write in "CREDITS FOR JAIL DAYS SERVED"? Also does not have to be all caps if preferred. :) Maybe the former to match with the ICC days.